uberfire-m2repo-editor/uberfire-m2repo-editor-backend/src/main/java/org/guvnor/m2repo/backend/server/helpers/HttpGetHelper.java

```diff
@@ -163,14 +163,10 @@ public void handle(final HttpServletRequest request,
         response.setDateHeader("Expires",
                                System.currentTimeMillis() + DEFAULT_EXPIRE_TIME);
 
-        RandomAccessFile input = null;
         OutputStream output = null;
-
-        try {
-            input = new RandomAccessFile(file,
-                                         "r");
+        try (RandomAccessFile input = new RandomAccessFile(file,
```

Comment:
Maybe the OutputStream below needs to be handled in try-with-resources too?

Reply:
No because that would make `output` final and is is being reused here, but maybe I can rewrite it :thinking: